samples/sample_encode/src/sample_encode.cpp

```diff
@@ -943,23 +975,24 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
     if (MFX_CODEC_MPEG2 != pParams->CodecId &&
         MFX_CODEC_AVC != pParams->CodecId &&
         MFX_CODEC_JPEG != pParams->CodecId &&
-        MFX_CODEC_HEVC != pParams->CodecId)
+        MFX_CODEC_HEVC != pParams->CodecId && 
+		MFX_CODEC_VP9 != pParams->CodecId)
```

Comment:
alignment is wrong

Reply:
alignment is correct - tabs displayed incorrectly here